common/constants.js

```diff
@@ -148,6 +148,9 @@ module.exports = Object.freeze({
     HUMAN_INTERVAL: 'human-readable-interval',
     CRON_EXPRESSION: 'cron_expression'
   },
+  CONFIG: {
```

Comment:
better to rename as `CONFIG_KEYS`

Reply:
Better not to rename this, as the value under is not a key but a prefix to the key. Also anything else related to config can be put under this `CONFIG` key. Ex., RESOURCE_NAME can be moved from `CONFIG_MAP` to here & rename `RESOURCE_NAME` to `SF_CONFIG`

Reply:
ok.. makes sense..